packages/headless/src/features/facets/range-facets/date-facet-set/date-facet-actions.ts

```diff
@@ -105,6 +115,7 @@ const dateFacetRegistrationOptionsDefinition = {
   injectionDepth: new NumberValue({required: false, min: 0}),
   numberOfValues: new NumberValue({required: false, min: 1}),
   sortCriteria: new Value<RangeFacetSortCriterion>({required: false}),
+  rangeAlgorithm: new Value<RangeFacetRangeAlgorithm>({required: false}),
```

Comment:
Why not use `StringValue` to validate `rangeAlgorithm` and `sortCriteria`?

Reply:
The `Value<RangeFacetRangeAlgorithm>` and `Value<RangeFacetSortCriterion>` force the value to be only from a selected group of values. For example, the type `RangeFacetRangeAlgorithm` can only be `even` or `equiprobable`. If another value not part of this selection is passed, it will throw an error to inform the user that he is passing an invalid value.